.github/docker/Dockerfile.base

```diff
@@ -1,14 +1,16 @@
-FROM rust:slim-bookworm AS builder
-RUN apt-get update && apt-get -y install curl
+FROM --platform=linux/amd64 rust:slim-bookworm AS amd64-tools
+RUN apt-get update && apt-get install -y --no-install-recommends curl bash
 RUN curl -L https://risczero.com/install | bash
 RUN /root/.risc0/bin/rzup install r0vm 2.1.0
 
-FROM ubuntu:noble
-COPY --from=builder /usr/local/cargo/bin/r0vm /usr/local/bin/r0vm
+# Barretenberg CLI
+RUN curl -L https://raw.githubusercontent.com/AztecProtocol/aztec-packages/refs/heads/master/barretenberg/bbup/install | bash \
+ && install -m 755 /root/.bb/bbup /usr/local/bin/bbup \
+ && bbup -v 0.82.2
```

Comment:
this command `bbup -v 0.82.2` installs barrenteberg to `/root/.bb` that  needs to be copied in final image in order to have `bb` binary available